### ISSUE TITLE
Error: no method named `encode` found for struct `actix_http::ws::Codec` in the current scope

### ISSUE THREAD

**Issue by Oyelowo** [async-graphql/async-graphql#824]
First, you've done a great job with this library and it's by far my favourite graphql library in any language I've worked with. Thanks a million for this!

## My question/Issue
I'm getting the error below while compiling my project. This started yesterday and I'm suspecting that it's from `async-graphql-actix-web` which uses the `actix-web-actors` [dependency](https://github.com/async-graphql/async-graphql/blob/master/integrations/actix-web/Cargo.toml).
It looks like [`actix-web-actor` was also updated yesterday](https://github.com/actix/actix-web/blob/master/actix-web-actors/Cargo.toml)

What could I be missing?

These are my dependencies:
```
# actix-rt = {version = "2.4.0", default-features = false, optional = true}
actix-web = {version = "4.0.0-rc.3", default-features = false, features = ["macros"]}
async-graphql = {version = "3.0.30", features = [
  "apollo_tracing",
  "apollo_persisted_queries",
  "bson",
  "dataloader",
  "chrono",
  "chrono-tz",
  "log",
  "dataloader",
  "tracing",
  "opentelemetry",
  "unblock",
  "url",
  "uuid",
  "string_number",
  "secrecy",
  "decimal",
  "smol_str",
]}
async-graphql-actix-web = "3.0.30"
tokio = {version = "1.16.1", features = ["full"]}
futures = "0.3"

mongodb = {version = "2.1.0", features = ["bson-chrono-0_4"]}
bson = "2.1.0"
# chrono = "0.4.19"
chrono = {version = "0.4.19", features = ["serde"]}
chrono-tz = "0.6.1"
# mongodb = "2.1.0"
serde = {version = "1.0", features = ["derive"]}
serde_json = "1.0"
slab = "0.4.5"
typed-builder = "0.9.1"
url = "2.2.2"
uuid = "0.8.2"
validator = {version = "0.12", features = ["derive"]}
wither = {git = "https://github.com/thedodd/wither", branch = "master"}
# wither = "0.9.0"

urlencoding = "2.1.0"
anyhow = "1.0.49"
envy = "0.4.2"
secrecy = "0.8.0"
thiserror = "1.0.30"
log = "0.4.14"

async-trait = "0.1.52"
config = "0.11.0"
crossbeam = "0.8.1"
derive_more = "0.99.17"
parking_lot = "0.11.2"
rayon = "1.5.1"
ron = "0.7.0"
serde-aux = "3.0.1"
# ts-rs = "6.1"
ts-rs = {version = "6.1", features = ["uuid-impl", "chrono-impl", "serde-compat"]}
```

## The error message

```
.....
#13 166.5    Compiling async-graphql-derive v3.0.30
#13 167.2    Compiling actix-http v3.0.0-rc.3
#13 167.3    Compiling tonic-build v0.6.2
#13 170.5    Compiling grpc-mongo v0.0.1 (/app/grpc-mongo)
#13 171.5    Compiling hyper v0.14.16
#13 172.0    Compiling actix-web v4.0.0-rc.3
#13 178.3    Compiling actix-web-actors v4.0.0-beta.11
#13 184.9 error[E0599]: no method named `encode` found for struct `actix_http::ws::Codec` in the current scope
#13 184.9    --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/actix-web-actors-4.0.0-beta.11/src/ws.rs:622:30
#13 184.9     |
#13 184.9 622 |                 this.encoder.encode(msg, &mut this.buf)?;
#13 184.9     |                              ^^^^^^ method not found in `actix_http::ws::Codec`
#13 184.9     |
#13 184.9    ::: /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/tokio-util-0.7.0/src/codec/encoder.rs:24:8
#13 184.9     |
#13 184.9 24  |     fn encode(&mut self, item: Item, dst: &mut BytesMut) -> Result<(), Self::Error>;
#13 184.9     |        ------ the method is available for `actix_http::ws::Codec` here
#13 184.9     |
#13 184.9     = help: items from traits can only be used if the trait is in scope
#13 184.9 help: the following trait is implemented but not in scope; perhaps add a `use` for it:
#13 184.9     |
#13 184.9 3   | use tokio_util::codec::encoder::Encoder;
#13 184.9     |
#13 184.9 
#13 185.0 error[E0599]: no method named `decode` found for mutable reference `&mut actix_http::ws::Codec` in the current scope
#13 185.0    --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/actix-web-actors-4.0.0-beta.11/src/ws.rs:705:28
#13 185.0     |
#13 185.0 705 |         match this.decoder.decode(this.buf)? {
#13 185.0     |                            ^^^^^^ method not found in `&mut actix_http::ws::Codec`
#13 185.0     |
#13 185.0     = help: items from traits can only be used if the trait is in scope
#13 185.0 help: the following trait is implemented but not in scope; perhaps add a `use` for it:
#13 185.0     |
#13 185.0 3   | use tokio_util::codec::decoder::Decoder;
#13 185.0     |
#13 185.0 
#13 185.2 For more information about this error, try `rustc --explain E0599`.
#13 185.3 error: could not compile `actix-web-actors` due to 2 previous errors
#13 185.3 warning: build failed, waiting for other jobs to finish...
#13 207.5 error: build failed
#13 207.7 thread 'main' panicked at 'Exited with status code: 101', /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/cargo-chef-0.1.33/src/recipe.rs:145:27
#13 207.7 note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c cargo chef cook --recipe-path recipe.json]: exit code: 101

```

**Comment by sunli829** [comment]
Fixed in `v3.0.31`

**Comment by Oyelowo** [comment]
Thanks for the swift response! All good now
